### DISCUSSION TITLE
Get Enum value associated with tiles

### DISCUSSION THREAD

**Discussion by Ggcpp** [Trouv/bevy_ecs_ldtk#314] [Q&A] [answered]
Hi! In my ldtk project I made an enum "collider" that defines different shapes of collider for my tiles. So I bound those different collider shapes to different tiles in my spritesheet but I don't know how to access those bindings in my code:

```main.rs
app.register_ldtk_int_cell_for_layer::<CollisionTileBundle>("Collisions", 1)

#[derive(Default, Component)]
struct CollisionTile;

#[derive(Default, Bundle, LdtkIntCell)]
struct CollisionTileBundle {
    collision_tile: CollisionTile,
}

fn process_collider_tiles(
    mut commands: Commands,
    collider_tiles: Query<Entity, Added<CollisionTile>>,
) {
    // How to know what type of collision shape has a tile? (by the bound enum value)
    // To add a corresponding collider component to the tiles.
}
```

How can I do that? Maybe my approach is just wrong?
Please let me know if you didn't understand something about my explanations :)

**Comment by Trouv** [comment, marked as answer]
The plugin should automatically spawn a component called `TileEnumTags` on your tile entities in this scenario: https://trouv.github.io/bevy_ecs_ldtk/latest/explanation/anatomy-of-the-world.html#tile-metadata-components

So you should be able to access that component in your `collider_tiles` query

**Reply by Ggcpp** [reply]
Ohh okay thanks!
